res/css/views/elements/_StyledPollCheckbox.pcss

```diff
@@ -0,0 +1,126 @@
+/*
+Copyright 2022 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/**
+* This component expects the parent to specify a positive padding and
+* width.
+* It is used for multiple selection polls and mostly copied
+* from _StyledRadioButton.pcss to match it's style.
+*/
+
```

Comment:
These are the same styles used by the radio button, so the checkbox could also use the classes from the `_StyledRadioButton.pcss` file which would need a single additional line to include the `input[type="checkbox"] selector (see screenshot). I'm not sure if that would impede that file's reusability or break the component's independence, so I chose to copy the styles into a new file.
<img width="538" alt="Bildschirmfoto 2022-11-14 um 16 15 00" src="https://user-images.githubusercontent.com/94838646/201696187-8739b71c-5c23-4227-8d02-404acbb9fc0f.png">

Reply:
I would merge the styles.